src/components/dropdownSearch/DropdownSearch.tsx

```diff
@@ -254,6 +266,10 @@ export class DropdownSearch extends React.Component<IDropdownSearchProps, {}> {
     return (elTop >= boxTop) && (elBottom <= boxBottom);
   }
 
+  private isSearchOn(): boolean {
+    return this.props.options.length > this.props.searchThresold;
```

Comment:
I've made the comment previously, but haven't had any feedback on it. What about when length is smaller than the threshold, but we can add a new option?

Also can you explain what this does? It is not quite clear when looking at the code.

Reply:
Sorry not having telling you, but if you look at the code and UTs, this is handled now @sroy3 .